PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -44,8 +44,8 @@ internal class PayPalInternalClient(
                 val url = "/v1/$endpoint"
                 val appLinkReturn = if (isBillingAgreement) appLink else null
 
-                if (isBillingAgreement) {
-                    (payPalRequest as PayPalVaultRequest).enablePayPalAppSwitch = isPayPalInstalled(context)
+                if (isBillingAgreement && (payPalRequest as PayPalVaultRequest).enablePayPalAppSwitch) {
```

Comment:
the feature could be turned off in settings, but during this validation, we would set `enablePayPalAppSwitch` to true if the Venice app was installed, which caused an odd flow

Reply:
Should we also check if the email is present here or are we doing that check elsewhere?

Reply:
We check that [here](https://github.com/braintree/braintree_android/blob/2b7ff747e520c231bc57560702d000b4f920034b/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.kt#L90)